src/backend/web/routes/index.js

```diff
@@ -10,6 +10,7 @@ const planet = require('./planet');
 const posts = require('./posts');
 const login = require('./login');
 const stats = require('./stats');
+const log = require('./log');
```

Comment:
Let's put this behind the `/admin/log` route instead of exposing it at the top level.